index.js

```diff
@@ -130,9 +130,7 @@ HtmlWebpackPlugin.prototype.emitHtml = function(compilation, htmlTemplateContent
   // Minify the html output
   if (this.options.minify) {
     var minify = require('html-minifier').minify;
-    // If `options.minify` is set to true use the default minify options
-    var minifyOptions = _.isObject(this.options.minify) ? this.options.minify : {};
-    html = minify(html, minifyOptions);
+    html = minify(html, this.options.minify);
```

Comment:
IDK if this change is necessary, as it might break someone's stuff. Passing in an empty object if it was just `true` before did nothing to the output though, so I'm guessing people using the `minify`-option do pass an object